plaid-stl/src/github/mod.rs

```diff
@@ -29,6 +29,23 @@ pub struct RepositoryCollaborator {
     pub permissions: Permission,
 }
 
+#[derive(Debug, Deserialize)]
+/// A person assigned to a seat in a Github Org Copilot subscription
+pub struct CopilotAsignee {
+    pub login: String,
+    #[serde(rename = "type")]
+    pub type_: String,
+}
+
+#[derive(Debug, Deserialize)]
+/// A seat in a Github Org Copilot subscription
+pub struct CopilotSeat {
+    pub assignee: CopilotAsignee,
+    pub last_activity_at: String,
```

Comment:
Should we parse this?

```
      "last_activity_at": "2021-10-14T00:53:32-06:00",
```

Reply:
I wouldn't. Let the rules parse it if they need it. Otherwise we introduce a new time dependency to everyone wanting to use the API without any knowledge of them needing it